README.md

```diff
@@ -1,7 +1,9 @@
 # pmdtester
 
-code  :: https://github.com/pmd/pmd-regression-tester  
-bugs  :: https://github.com/pmd/pmd-regression-tester/issues  
+code  :: https://github.com/pmd/pmd-regression-tester
+
+bugs  :: https://github.com/pmd/pmd-regression-tester/issues
+
 
```

Comment:
We can add here the badge from travis:

```
[![Build Status](https://travis-ci.com/pmd/pmd-regression-tester.svg?branch=master)](https://travis-ci.com/pmd/pmd-regression-tester)
```

It shows as: [![Build Status](https://travis-ci.com/pmd/pmd-regression-tester.svg?branch=master)](https://travis-ci.com/pmd/pmd-regression-tester)

Reply:
I know now, what the double colons mean: It's not markdown, it's the ruby documentation format...
I guess, we should rename the file to be `README.rdoc`

The build status line looks then like this:

```
build-status :: {<img src="https://travis-ci.com/pmd/pmd-regression-tester.svg?branch=master" alt="Build Status" />}[https://travis-ci.com/pmd/pmd-regression-tester]
```

And remove the extra line you introduce between code and bugs. The double colon makes this a nice table, see e.g. https://github.com/ruby/rake/blob/master/README.rdoc

Note: the headings are also different, you can see the syntax here: https://raw.githubusercontent.com/ruby/rdoc/master/ExampleRDoc.rdoc